package-lock.json

```diff
@@ -9,7 +9,7 @@
       "version": "3.7.0",
       "license": "MIT",
       "devDependencies": {
-        "@tacc/core-styles": "git+https://git@github.com/TACC/Core-Styles.git#v0.5.0",
+        "@tacc/core-styles": "https://gitpkg.now.sh/TACC/tup-ui/libs/core-styles?tup-starter-nx--core-styles-integration",
```

Comment:
Hmm, I'm hesitant about depending on some external host for this. How is it better than what you're replacing here?

Reply:
I am also hesitant. Don't approve 😅.

This manner of install is not better. It is a placeholder solution to migrate to [tup-ui](https://github.com/TACC/tup-ui) monorepo until I publish `@tacc/core-styles` on NPM (coming soon, maybe today) (at which point I can write `"@tacc/core-styles": "^0.5.1"` instead of this).

<details><summary>We talked about this new monorepo in our meeting, but I'll answer for posterity (click to reveal).</summary>

The [tup-ui](https://github.com/TACC/tup-ui) monorepo can be considered better because it allows live-edit of TUP UI and Core Styles and Core Components simultaneously without using `npm link`. (Using `npm link` for Core-Portal is documented in #639.) I do not think a monorepo.

</details>

Reply:
(I edit the hidden answer a lot since posting, so take another read if you already read it earlier.)

Reply:
Okay. I published [`@tacc/core-styles` on NPM](https://www.npmjs.com/package/@tacc/core-styles). Expect an update to this PR (and https://github.com/TACC/Core-Portal/pull/651) later.

Reply:
Now (965ebfe) Core Styles is installed via NPM at `v0.6.0-alpha` (which is published from [tup-ui:/libs/core-ui](https://github.com/TACC/tup-ui/tree/tup-starter-nx--core-styles-integration)).